onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -474,7 +474,9 @@
         "^test_resize_downsample_scales_linear", // Runs but there's accuracy mismatch
         "^test_training_dropout_default_mask", // Runs but there's accuracy mismatch
         "^test_training_dropout_mask", // Runs but there's accuracy mismatch
-        "^test_training_dropout_default" // Runs but there's accuracy mismatch
+        "^test_training_dropout_default", // Runs but there's accuracy mismatch
+        "^test_center_crop_pad_crop_negative_axes_hwc*", // failed because new types added for CenterCropPad
+        "^test_center_crop_pad_crop_negative_axes_hwc_expanded*" // failed because new types added for CenterCropPad
```

Comment:
IIRC, recently ONNX fixed negative axes support for CenterCropPad and also these two tests were brought: https://github.com/onnx/onnx/pull/5007. I wonder what the exact error is that onnxruntime bumps into with these two?

Reply:
good catch! however this failure occurred after rc1 so I inferred that must be due to changes after rc1. The error message is not specific enough to figure out exactly the cause. Let add this possibility to the comment. 
I tried to build with OpenVino, but I am running out of time. Need someone from OpenVino to investigate.